### DISCUSSION TITLE
Old livery compatibility

### DISCUSSION THREAD

**Discussion by TheHi-Champion** [magknight/787aviatorsTracker#635] [Q&A] [answered]
Make all liveries that are older compatible with newer magknight updates, I would love that for being able to have capability to use any livery that exists to the max.

**Comment by aeroniemi** [comment, marked as answer]
Any livery released since 1.1.0 is still compatible, providing it was shipped with a valid liveryConfig file at the time - if it wasn't, the plane has no way of knowing what art the livery wants to use.
